docs/experiments/categories/gcp/gcp-vm-instance-stop/auto-scaling.yaml

```diff
@@ -0,0 +1,30 @@
+## scale up and down to maintain the available instance counts
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
```

Comment:
Can we keep the name as `gcp-vm-chaos`, as that is what we have mentioned in the docs as default?